content/blog/2020-05-14-may-20-heartbeat.md

```diff
@@ -0,0 +1,180 @@
+---
+title: May '20 DVC❤️Heartbeat
+date: 2020-05-14
+description: |
+  Catch up on new DVC releases, talks, and projects in our community. 
+  This month, learn what we're up to in MLOps, CI/CD, and the 
+  intersection of data science and software engineering.
+
+descriptionLong: |
+  Every month we share news, findings, interesting reads, community takeaways,
+  and everything else along the way.
+
+  Look here for updates about [DVC](https://dvc.org), our journey as a startup,
+  projects by our users and big ideas about best practices in ML and data
+  science.
+picture: 2020-05-14/May_20_Heartbeat.png
+pictureComment: A big hello from DVC mascot DeeVee.
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/april-20-heartbeat/347
+tags:
+  - Heartbeat
+  - Plots
+  - MLOps
+  - Meetup
+  - Season of Docs
```

Comment:
`Google Season of Docs`- https://dvc.org/blog/tags/google-season-of-docs

Reply:
fixed